### PR TITLE
Iain/special report alt designs

### PR DESCRIPTION
**Pull request by iainjchambers-guardian**
## What does this change?
Adds special report alt designs using the new corresponding colour palette

## Screenshots
| Light Mode | Dark Mode |
| --- | --- |
| image | image |
| image | image |
| image | image |
| image | image |

[before]: https://example.com/before.png
[after]: https://example.com/after.png

### CODE REVIEW

File: apps-rendering/src/components/Layout/Layout.stories.tsx [hunk 1]
```diff
@@ -61,6 +63,21 @@ export const Article = (): React.ReactNode => {
...
+				formatFromItem(
+					{ ...article, theme: 7 },
+					some(ArticleDisplay.Standard),
+				),
+				Result.partition(article.body).oks,
+			)}
+		</Standard>
+	);
+};
+ArticleSRA.story = { name: 'Article - SRA' };
```
Comment (georgeblahblah) [inline apps-rendering/src/components/Layout/Layout.stories.tsx:79]: Just for clarity! Took me a second to expand the acronym

```suggestion
ArticleSRA.story = { name: 'Article - Special Report Alt' };
```

File: apps-rendering/src/components/Layout/Layout.stories.tsx [hunk 2]
```diff
@@ -169,6 +186,34 @@ export const CommentItem = (): React.ReactNode => {
 };
 CommentItem.story = { name: 'Comment' };
 
+export const CommentItemSRA = (): React.ReactNode => {
+	return (
+		<Comment
+			item={{
+				...comment,
+				theme: 7,
```
Comment (georgeblahblah) [inline apps-rendering/src/components/Layout/Layout.stories.tsx:194]: Could we use the enum field here?

File: apps-rendering/src/components/Layout/Layout.stories.tsx [hunk 3]
```diff
@@ -169,6 +186,34 @@ export const CommentItem = (): React.ReactNode => {
...
+				series: some<Tag>({
+					id: '',
+					type: TagType.SERIES,
+					webTitle: 'Cottonopolis',
```
Comment (georgeblahblah) [inline apps-rendering/src/components/Layout/Layout.stories.tsx:198]:
```suggestion
					webTitle: 'Special Report Alt',
```

File: apps-rendering/src/components/Layout/Layout.stories.tsx [hunk 4]
```diff
@@ -169,6 +186,34 @@ export const CommentItem = (): React.ReactNode => {
...
+					webUrl: '',
+					apiUrl: '',
+					references: [],
+				}),
+			}}
+		>
+			{renderAll(
+				formatFromItem(
+					{ ...comment, theme: 7 },
```
Comment (georgeblahblah) [inline apps-rendering/src/components/Layout/Layout.stories.tsx:207]: Could we use the enum field here as well?

File: apps-rendering/src/components/Layout/Layout.stories.tsx [hunk 5]
```diff
@@ -252,7 +312,31 @@ export const Immersive = (): ReactElement => (
...
+export const ImmersiveSRA = (): ReactElement => (
+	<ImmersiveLayout
+		item={{
+			...immersive,
+			edition: Edition.UK,
+			theme: 7,
+			series: some<Tag>({
+				id: '',
+				type: TagType.SERIES,
+				webTitle: 'Cottonopolis',
```
Comment (georgeblahblah) [inline apps-rendering/src/components/Layout/Layout.stories.tsx:326]:
```suggestion
				webTitle: 'Special Report Alt',
```

File: apps-rendering/src/components/Layout/Layout.stories.tsx [hunk 6]
```diff
@@ -252,7 +312,31 @@ export const Immersive = (): ReactElement => (
...
 );
-Immersive.story = { name: 'Immersive ' };
+Immersive.story = { name: 'Immersive' };
+
+export const ImmersiveSRA = (): ReactElement => (
+	<ImmersiveLayout
+		item={{
+			...immersive,
+			edition: Edition.UK,
+			theme: 7,
```
Comment (georgeblahblah) [inline apps-rendering/src/components/Layout/Layout.stories.tsx:322]: Enum field again!

File: apps-rendering/src/components/Layout/Layout.stories.tsx [hunk 7]
```diff
@@ -252,7 +312,31 @@ export const Immersive = (): ReactElement => (
...
+				type: TagType.SERIES,
+				webTitle: 'Cottonopolis',
+				webUrl: '',
+				apiUrl: '',
+				references: [],
+			}),
+		}}
+	>
+		{renderAll(
+			{ ...formatFromItem(immersive, none), theme: 7 },
```
Comment (georgeblahblah) [inline apps-rendering/src/components/Layout/Layout.stories.tsx:334]: Enum field again for this theme!

File: apps-rendering/src/components/Layout/Layout.stories.tsx [hunk 8]
```diff
@@ -252,7 +312,31 @@ export const Immersive = (): ReactElement => (
...
+			Result.partition(immersive.body).oks,
+		)}
+	</ImmersiveLayout>
+);
+ImmersiveSRA.story = { name: 'Immersive - SRA' };
```
Comment (georgeblahblah) [inline apps-rendering/src/components/Layout/Layout.stories.tsx:339]:
```suggestion
ImmersiveSRA.story = { name: 'Immersive - Special Report Alt' };
```

File: apps-rendering/src/components/Layout/Layout.stories.tsx [hunk 9]
```diff
@@ -205,6 +250,21 @@ export const Analysis = (): React.ReactNode => {
 };
 Analysis.story = { name: 'Analysis' };
 
+export const AnalysisSRA = (): React.ReactNode => {
+	return (
+		<AnalysisLayout item={{ ...analysis, theme: 7 }}>
+			{renderAll(
+				formatFromItem(
+					{ ...analysis, theme: 7 },
```
Comment (georgeblahblah) [inline apps-rendering/src/components/Layout/Layout.stories.tsx:258]: enum field!

File: apps-rendering/src/components/Layout/Layout.stories.tsx [hunk 10]
```diff
@@ -61,6 +63,21 @@ export const Article = (): React.ReactNode => {
 };
 Article.story = { name: 'Article' };
 
+export const ArticleSRA = (): React.ReactNode => {
+	return (
+		<Standard item={{ ...article, theme: 7 }}>
+			{renderAll(
+				formatFromItem(
+					{ ...article, theme: 7 },
```
Comment (georgeblahblah) [inline apps-rendering/src/components/Layout/Layout.stories.tsx:71]: enum field!

File: apps-rendering/src/components/Layout/Layout.stories.tsx [hunk 11]
```diff
@@ -61,6 +63,21 @@ export const Article = (): React.ReactNode => {
 };
 Article.story = { name: 'Article' };
 
+export const ArticleSRA = (): React.ReactNode => {
+	return (
+		<Standard item={{ ...article, theme: 7 }}>
```
Comment (georgeblahblah) [inline apps-rendering/src/components/Layout/Layout.stories.tsx:68]: enum field!

**Review by JamieB-gu — COMMENTED** [review]
Some great stuff here 🚀.

I have a few questions and comments, some of which may be for @HarryFischer. I've broken them up into sections.

Some or all of these suggestions could happen in a follow-up PR.

## Articles For Which We Don't Have Custom Designs

Although we don't have custom Special Report Alt designs for every kind of article, all articles should still work if they have that theme applied. I've gathered screenshots below of how some `Design` variants currently look with these changes, and some suggestions for how I think these should probably look. @HarryFischer let us know if you have any other suggestions.

| `Design` | Gallery | Deadblog | Liveblog | Letters |
| - | - | - | - | - |
| Screenshot | ![gallery] | ![deadblog] | ![liveblog] | ![letters] |
| Suggested design | Normal gallery (dark background, light text) | News deadblog | News liveblog | `Comment` special report alt design |

[letters]: https://user-images.githubusercontent.com/53781962/203396431-c8d4748e-b98b-482d-b80a-93fa067216e9.jpg
[liveblog]: https://user-images.githubusercontent.com/53781962/203396438-4491b828-de6d-406e-8260-54aa60ed72a9.jpg
[gallery]: https://user-images.githubusercontent.com/53781962/203396441-59872b08-aca2-4fb9-a250-bc79d0982f9e.jpg
[deadblog]: https://user-images.githubusercontent.com/53781962/203396443-e52ea1ca-1be1-4c5c-83df-80188e1e6e81.jpg

## Contrast

There are a few places where I think we need to tweak contrast to meet requirements.

### Bylines

I think on the designs these are `specialReportAlt.200` for light and `specialReportAlt.300` for dark, so it should just be a case of updating to match?

| `Design` | Light | Dark |
| - | - | - |
| Comment | ![byline-comment] | ![byline-comment-dark] |
| Standard | ![byline-standard] | ![byline-standard-dark] |

[byline-standard]: https://user-images.githubusercontent.com/53781962/203399725-f2ada4d3-c4a1-40e5-a499-d372b9d58431.jpg
[byline-comment-dark]: https://user-images.githubusercontent.com/53781962/203399727-bc786182-4827-4c73-ae0e-e426bdbe0dc8.jpg
[byline-comment]: https://user-images.githubusercontent.com/53781962/203399733-4cc5719d-9d3d-4074-b1a7-ee3af6b5ba3c.jpg
[byline-standard-dark]: https://user-images.githubusercontent.com/53781962/203399737-a1c61fab-178b-4a6c-ae5a-4fb79b83f74b.jpg

### Dateline And Captions

The colour for these seems to vary in the designs, so perhaps @HarryFischer can specify which we should use? I'm assuming both can be given the same colour, as they're both sans and the same font size?

| Dateline | Caption |
| - | - |
| ![byline-standard] | ![caption] |

[caption]: https://user-images.githubusercontent.com/53781962/203401625-7244bcb1-5098-4366-a18f-865ab3d6d714.jpg

### Shapes (Lines, Borders, Link Underlines, Bullets)

@HarryFischer perhaps you could provide some rules on what colours we should use for these (light and dark)? The bullets and link underlines in particular seem very light on this new background.

## Interview Headline

@HarryFischer interviews normally have a highlight on the headline. Is that something we'd want to keep in these new designs?

| News | Special Report Alt |
| - | - |
| ![interview-standard] | ![interview-special] |

[interview-standard]: https://user-images.githubusercontent.com/53781962/203403395-ec65b839-addf-4cd6-8b40-7775d1f9c859.jpg
[interview-special]: https://user-images.githubusercontent.com/53781962/203403399-b89099a0-bdac-47a5-927b-b91801eeda65.jpg

**Pull request closed without merging**